src/GitVersion.Core.Tests/BuildAgents/GitHubActionsTests.cs

```diff
@@ -83,7 +83,7 @@ public void GetCurrentBranchShouldHandleTags()
         var result = this.buildServer.GetCurrentBranch(false);
 
         // Assert
-        result.ShouldBe("refs/tags/1.0.0");
```

Comment:
Similar assertions in other build server tests:

https://github.com/GitTools/GitVersion/blob/666bd37b59ae731b1d2bd40b20364b692b21796e/src/GitVersion.Core.Tests/BuildAgents/SpaceAutomationTests.cs#L74